mozregression/fetch_configs.py

```diff
@@ -222,11 +222,15 @@ class InboundConfigMixin(object):
     Define the inbound-related required configuration.
     """
     inbound_branch = 'mozilla-inbound'
+    branch_path = 'integration'
```

Comment:
Where is this variable actually used? I don't see anything functional in this patch actually referring to `branch_path`.